plugins/modem-manager/fu-cinterion-fdl-updater.c

```diff
@@ -0,0 +1,349 @@
+/*
+ * Copyright (C) 2024 TDT AG
+ *                    Lukas Voegl <lvoegl@tdt.de>
```

Comment:
needs to be on one line, e.g. `Copyright 2020 Richard Hughes <richard@hughsie.com>`